### PR TITLE
Fix #165624 check-host.net

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

> I figured they might try and remove https as a way to bypass it, and they did, so I hope this new rule is okay. I did test it, and it does not break anything.

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

check-host.net: <https://github.com/AdguardTeam/AdguardFilters/issues/165624>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by xglossyntx** [comment]
Also this ad seems to be back 
image

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -278,7 +278,7 @@ $script,third-party,denyallow=fastly.net|statically.io|sharecast.ws|bunnycdn.ru|
 ! NOTE: Specific rules
 !
 check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+)$/):upward(2)
-check-host.net##a[href^="https://specialrdp.com/"]
+check-host.net##a[href*="specialrdp.com"]
 /^https:\/\/check-host\.net\/images\/[0-9]?_?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:282]: Also exchange current rules to these ones.
```adblock
check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋)$/):upward(2)
/^https:\/\/check-host\.net\/images\/[0-9]?[a-z]?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]
Nvm, I've done it in the branch.

**Comment by xglossyntx** [comment]
Sorry about that. still getting used to navigating GitHub.

**Comment by zloyden** [comment]
Don't worry, you did well:)

**Pull request merged**